### PR TITLE
Arrows Stuck

### PR DESCRIPTION
**Pull request by Wealthyturtle**
### Description
This PR allows users to set/get the number of arrows stuck in any LivingEntity. Why not?

---
**Target Minecraft Versions:**     Any (I think - methodExists() is used)
**Requirements:**     Paper
**Related Issues:**     #1655

### CODE REVIEW

File: src/main/java/ch/njol/skript/expressions/ExprArrowsStuck.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+@Name("Arrows Stuck")
+@Description("The number of arrows stuck in a living entity.")
+@Examples("set player's stuck arrows to 5")
+@Since("INSERT VERSION")
+public class ExprArrowsStuck extends SimplePropertyExpression<LivingEntity, Number> {
+
+    static {
+    	if (Skript.methodExists(LivingEntity.class, "getArrowsStuck")) {
+    		register(ExprArrowsStuck.class, Number.class, "stuck arrow[s]", "livingentities");
```
Comment (Pikachu920) [inline src/main/java/ch/njol/skript/expressions/ExprArrowsStuck.java]: wouldn't it be nice to have `add 1 to arrows stuck in player`? this syntax doesn't allow that

Comment (Pikachu920) [reply src/main/java/ch/njol/skript/expressions/ExprArrowsStuck.java]: in fact i'm not sure if `arrows stuck of player` makes sense at all

Comment (Wealthyturtle) [reply src/main/java/ch/njol/skript/expressions/ExprArrowsStuck.java]: Hmm... that's a good point. Will probably not be able to use a SPE for this expression.

Comment (Pikachu920) [reply src/main/java/ch/njol/skript/expressions/ExprArrowsStuck.java]: you can use an SPE (and should), you just won't be able the prebuilt registration method

Comment (Wealthyturtle) [reply src/main/java/ch/njol/skript/expressions/ExprArrowsStuck.java]: Hopefully the new pattern is better.

**Comment by Pikachu920** [comment]
what if the changer accepted Arrow itemstacks? like `add 5 arrows to arrows stuck in player`? i think that would be pretty nice, not to mention it's simple to add

**Comment by Wealthyturtle** [comment]
Ehhhhhh I mean honestly I find it quite pointless - not sure why people would do that when they can just increment the number of arrows via the number.

**Comment by Pikachu920** [comment]
because `set arrows stuck in player to 10` reads rougher than `set arrows stuck in player to 10 arrows`

either way, what happens if you try to set it to 1000? that's far more than the max of a byte which is what this is stored as in NMS so I wonder if it throws an exception?

**Comment by Wealthyturtle** [comment]
I tested with 1000, no error or anything, and the entity's arrow stuck number is reported correctly as 1000.

**Comment by bensku** [comment]
How does MC client react to too many arrows?

**Comment by Wealthyturtle** [comment]
It just seems to stop rendering... not sure if that's an issue on my client though.

**Comment by Wealthyturtle** [comment]
Added a syntax test for this expression.

**Comment by Wealthyturtle** [comment]
Added the Remove ChangeMode, with proper handling for negative addition/negative removes.

**Pull request merged**